client/packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/TabTables.tsx

```diff
@@ -68,24 +75,36 @@ const getExpiryColumn = (
   },
 ];
 
-const NumberOfPacksCell: React.FC<CellProps<DraftInboundLine>> = ({
+const NumberOfPacksCell = ({
   rowData,
   ...props
-}) => (
+}: CellProps<DraftInboundLine>) => (
   <NumberInputCell
     {...props}
     isRequired={rowData.numberOfPacks === 0}
     rowData={rowData}
   />
 );
 
-export const QuantityTableComponent: FC<TableProps> = ({
+export const QuantityTableComponent = ({
   lines,
   updateDraftLine,
   isDisabled = false,
-}) => {
+  item,
+  hasLinkedShipment,
+}: TableProps) => {
+  const t = useTranslation();
   const theme = useTheme();
+  const { data: preferences } = usePreference(
+    PreferenceKey.DisplayVaccineInDoses
+  );
+  const { getColumnLabelWithPackOrUnit } = useIntlUtils();
   const itemVariantsEnabled = useIsItemVariantsEnabled();
+  const displayInDoses =
+    !!preferences?.displayVaccineInDoses && !!item?.isVaccine;
+  const shouldShowNumberOfPacks =
+    !displayInDoses || (displayInDoses && !hasLinkedShipment);
```

Comment:
Hm, I know this was in the designs (not show the num packs/packsize on automatic inbound) - and I think fine for lines that were transferred... but if I add a new item or new batch to a transferred inbound, it just uses default pack size - what if I want to set it? 

I.e. this item wasn't included in the transferred inbound, but I received it, I want to set pack size 10 but I can't? Uses the default of pack size 2 on save..:
<img width="1079" alt="Screenshot 2025-05-05 at 9 34 26 AM" src="https://github.com/user-attachments/assets/7ea306e1-7726-49eb-9d3f-607e636cc0df" />

Or is that not a valid request, if I am uses "display in doses", I don't care about pack size?? If that were the case, would expect pack size/num packs hidden on manual inbounds as well... @mark-prins

Reply:
According to @craigdrown, if a facility is handling vaccines in doses, they don't care about pack size... Asked the same question.

Though, maybe valid for `manually` added lines in an `automatically` transferred invoice? We do save `linked_invoice_id` now on invoice lines, so can handle case if you think it's required @mark-prins